pkg/plugins/intel/intel_plugin.go

```diff
@@ -44,3 +44,10 @@ func (p *IntelPlugin) Apply() error {
 	glog.Info("intel-plugin Apply()")
 	return nil
 }
+
+func (p *IntelPlugin) SetSystemdFlag() {
```

Comment:
`SetSystemdFlag`/`IsSystemService` pair seems to be unused. Isn't it?

Reply:
done